packages/client/src/App.sass

```diff
@@ -246,6 +250,10 @@ h2
 .button.is-small.with-icon>svg
   transform: scale(0.7)
 
+/********************* Custom utility css - Columns ********************/
+.column
```

Comment:
This will override the default column class, impacting every element with column class. Are we sure we want to do this? Is there an alternative way?

Reply:
removed this